### ISSUE TITLE
Add Siri integration

### ISSUE THREAD

**Issue by chadsy** [OneBusAway/onebusaway-iphone#815]
Add SiriKit integration for iOS 10+...

"Hey Siri, when is the next 550 to Seattle?"

"Hey Siri, do I have time to catch the bus at the library?"

"Hey Siri, remind me to catch the next 271 to the U District"

**Comment by aaronbrethorst** [comment]
Are transit apps supported by Siri in iOS 10? I thought they only had an intent for TNCs... That would be awesome if we could do it, though!

**Comment by chadsy** [comment]
Good point. I'd assumed Ride Booking would cover it. This will require some deeper thought, obvs...

**Comment by aaronbrethorst** [comment]
I'm all about if it works...

**Comment by ualch9** [comment]
Based on my experiences with SiriKit, it appears that Siri doesn't integrate well with a service such as OneBusAway. Siri requires constant updates on vehicle locations and the language presented to the user is definitely confusing. As of iOS 10.2.1, I would not recommend adding Siri extension.
![file](https://cloud.githubusercontent.com/assets/22162410/22234583/08080978-e1ae-11e6-83ea-00c327e06d61.jpeg)

**Comment by aaronbrethorst** [comment]
I need to do some more research, but it looks like iOS 12 may finally allow this to work! https://developer.apple.com/sirikit/
